requirements.txt

```diff
@@ -1,7 +1,7 @@
 aiohttp
 aiohttp_cors==0.7.0
 attr
-homeassistant>=2022.4.5
+homeassistant>=2022.6.1
```

Comment:
Is this needed for this behavior or might as well require latest version since this is a new major release?

Reply:
It's needed. The prior didn't have the hidden/visible behavior when I tried to use it, so just updated it.

Reply:
Oh yeah duh, sounds good